src/Assets/TestProjects/TestConsoleApp/ConsoleApp.csproj

```diff
@@ -0,0 +1,10 @@
+<Project Sdk="Microsoft.NET.Sdk">
+  <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory), testAsset.props))\testAsset.props" />
+
+  <PropertyGroup>
+    <OutputType>Exe</OutputType>
+    <TargetFramework>net6.0</TargetFramework>
+    <LangVersion>preview</LangVersion>
```

Comment:
Ignore this for now, I'm still figuring out why Roslyn changes for C# 10 still isn't working properly in this repo. This will be removed before this change is merged.

Reply:
https://github.com/dotnet/aspnetcore/pull/33393/files

Look for something pinning Microsoft.Net.Compilers.Toolset e.g. https://github.com/dotnet/sdk/blob/9f265a2db3d9c9bcd9c1ec0eeb7fc25f1a8f001b/eng/Versions.props#L120